docs/presentation_outline.txt

```diff
@@ -17,7 +17,7 @@ Managing State
 8. Sean
 
 Service
-9. Sean? Andy?
+9. Sean
```

Comment:
I won't be able to do this.

Reply:
👍 I wasn’t really thinking this through when I made the in-line change. Definitely didn’t mean to auto-volunteer you like that w/o consulting you.